README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build status](https://github.com/neofinancial/config-dug/workflows/CI/badge.svg)](https://github.com/neofinancial/ccscan/actions)
 ![TypeScript 3.5.3](https://img.shields.io/badge/TypeScript-3.5.3-brightgreen.svg)
+[![coverage](https://coverage.neotools.ca/api/badge/master/sU3MNGvvrQN9LyiFXr1uy3)](https://coverage.neotools.ca/coverage/neofinancial/ccscan)
```

Comment:
Not sure where to get the link for the codecov badge

Reply:
looks like you got this from the configuration should be good then?